utils/custom.js

```diff
@@ -1,7 +1,10 @@
+const {NOTIFICATION_TOPIC_NAME_PREFIX} = require('../helpers/constants');
+
 const emojiUnicode = require('emoji-unicode');
 const _ = require('lodash');
 
 const convertString = (str, from, to) => {
+  if (str === null || str === undefined) return str;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a null and undefined check in the `convertString` function is a good practice to prevent potential errors. However, it would be better to return an empty string instead of the input value when the input is null or undefined. This way, the function's return type will always be a string, which can help avoid unexpected behavior in other parts of the code that use this function.
```diff
- if (str === null || str === undefined) return str;
+ if (str === null || str === undefined) return '';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->